guides/v2.2/cloud/howtos/debug.md

```diff
@@ -10,24 +10,24 @@ functional_areas:
   - Setup
 ---
 
-We recommend using Xdebug for your PHP debugging. The following sections detail how to configure Xdebug and PhpStorm to work locally with settings in your environments. You can use any IDE of your choice. See the vendor documentation for those applications for further configuration information.
+Xdebug is an extension for debugging your PHP. The following explains how to configure Xdebug and PhpStorm to debug in your local environment. You can use any IDE of your choice. See the vendor documentation for those applications for further configuration information.
 
-To set up Xdebug, you need to [configure](#configure) a file in your Git repo, configure your IDE PhpStorm, and set up port forwarding. You will configure settings in the `magento.app.yaml` file. After editing, you can push the Git changes across all Starter environments and Pro Integration environments to enable Xdebug. To push these settings to Pro plan Staging and Production environments, you must enter a ticket.
+To set up Xdebug, you need to [configure](#configure-xdebug) a file in your Git repository, configure your IDE PhpStorm, and set up port forwarding. You can configure settings in the `magento.app.yaml` file. After editing, you can push the Git changes across all Starter environments and Pro Integration environments to enable Xdebug. To push these settings to Pro plan Staging and Production environments, you must enter a ticket.
 
-Once configured, you can debug [CLI commands](#debugcli), [web requests](#webrequests), [code](#code). Remember, all {{site.data.var.ece}} environments are read-only. You will need to pull code to your local development environment to perform debugging. For Pro plan Staging and Production, we include [additional instructions](#pro-debug) for Xdebug on those environments.
+Once configured, you can debug [CLI commands](#debugcli), [web requests](#webrequests), [code](#code). Remember, all {{site.data.var.ece}} environments are read-only. You need to pull code to your local development environment to perform debugging. For Pro Staging and Production environments, we include [additional instructions](#pro-debug) for Xdebug.
 
 ## Requirements {#usexdebug}
 
-To run and use Xdebug, you will need the environment's SSH URL. You can locate the information through the [Project Web Interface]({{ page.baseurl }}/cloud/project/projects.html) or your spreadsheet.
+To run and use Xdebug, you need the SSH URL for the environment. You can locate the information through the [Project Web Interface]({{ page.baseurl }}/cloud/project/projects.html) or your spreadsheet.
```

Comment:
```suggestion
To run and use Xdebug, you need the SSH URL for the environment. You can locate the information through the [Project Web Interface]({{ page.baseurl }}/cloud/project/projects.html) or your Cloud Onboarding UI.
```

Reply:
Onboarding spreadsheet not used anymore.  Now use Cloud Onboarding UI -- that's the name that CSM and TAMs use to refer to the interface created by Rainbow team.